docs/src/main/sphinx/connector/table-redirection.fragment

```diff
@@ -0,0 +1,65 @@
+In the context of connectors which depend on a metastore service
+(for example, :doc:`/connector/hive`, :doc:`/connector/iceberg` and :doc:`/connector/delta-lake`),
+the metastore (Hive metastore service, `AWS Glue Data Catalog <https://aws.amazon.com/glue/>`_)
+can be used to accustom tables with different table formats.
+Therefore, a metastore database can hold a variety of tables with different table formats.
```

Comment:
This is the motivation for initial usecases, but not the requirement. IMO we can swap this with the second paragraph. (i.e. first introduce the concept of redirection, and then say that it's currently implemented for connectors depending on metastore service)